vendor/plugins/edu.harvard.dce.paella.timedCommentsPlugin/timedCommentsOverlay.js

```diff
@@ -770,6 +776,15 @@ Class ("paella.plugins.TimedCommentsOverlay", paella.EventDrivenPlugin, {
     return parser.parseFromString(template, "text/html");
     // returns a HTMLDocument, which also is a Document.
   },
+  hasAdminRole: function(userRoles) {
+    var isAdmin = false;
+    this._adminRoles.forEach(function(role) {
+      if ($.inArray(role, userRoles) !== -1) {
```

Comment:
Just return here with no local variable?

Reply:
Yes. I'll make this change. (i.e. Array.some() vs Array.forEach)